.github/workflows/velox_backend.yml

```diff
@@ -527,7 +527,9 @@ jobs:
         run: |
           export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk && \
           cd /opt && \
+          ${WGET_CMD} https://archive.apache.org/dist/incubator/uniffle/${{ matrix.uniffle }}/apache-uniffle-${{ matrix.uniffle }}-incubating-bin.tar.gz && \
```

Comment:
This will download the package on each run and Apache archival site has a restriction on current download. Would you please improve this by doing a check if this file exists firstly?  
Cc: @PHILO-HE

Reply:
Oh, sorry for miss that. @SteNicholas could you add a followup and add the file check like [celeborn](https://github.com/apache/incubator-gluten/blob/main/.github/workflows/velox_backend.yml#L585)?

Reply:
@jackylee-ch, ok. I will add a followup.